pkg/attribute/bag.go

```diff
@@ -46,6 +40,9 @@ type Bag interface {
 
 	// Bytes returns the named attribute if it exists.
 	Bytes(name string) ([]uint8, bool)
+
+	// Close indicates the bag can be reclaimed.
+	Close()
```

Comment:
A method named Close should return error, by strong convention. I would suggest another name, like Done.